### DISCUSSION TITLE
HAT for Phoniebox

### DISCUSSION THREAD

**Discussion by stopfkuchen** [MiczFlor/RPi-Jukebox-RFID#2360] [Ideas]
Hi everyone,

I have developed a HAT that combines ON/OFF shim, Li-ION Batterypack Management and 2x3W Audio Amp on one board.
I developed it primarily for the MuPiBox, hovever it's working well with Phoniebox.
Maybe the community is interested, see mupihat.de [MupiHAT](https://mupihat.de/)

Cheers,
Lars

![IMG_0126](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/6362751/d535a25e-3dd0-4f63-a503-1ffab4ea8f27)

**Comment by s-martin** [comment]
Sounds interesting 🎉

**Comment by s-martin** [comment]
For reference how this needs to be configured: https://discord.com/channels/879874342203306005/932225388384489473/1234419040202657834

**Comment by stopfkuchen** [comment]
https://mupihat.de/products/mupihat-2

**Comment by pskiebe** [comment]
The MuPiHAT is really easy to adapt to something other than a MuPiBox, you should try it for the Phoniebox. I built this with it. It's not a Phoniebox, it's something completely different: [tonuinopi](https://gitlab.com/pskiebe/tonuinopi)
